rest_framework/filters.py

```diff
@@ -48,14 +48,15 @@ class SearchFilter(BaseFilterBackend):
     }
     search_title = _('Search')
     search_description = _('A search term.')
+    search_fields = None
```

Comment:
I have two concern here. 1. we probably need to document this enhancement 2. we need some tests to see this is actually working as it is described.

Reply:
Thank you, I take your comment as pre-approval.

As I indicated at the end of this PR description, I created this minimal PR to get feedback on whether it makes sense. The change is minimal (made by GitHub editor) and adding documentation and tests will take a bit longer, but I fully understand that it is required and I will provide than soon.